### ISSUE TITLE
Lambda ServiceException

### ISSUE THREAD

**Issue by lusentis** [mentum/lambdaws#49]
Hi,
 the example does not work for me, on the command line I see a "-->" and from CloudWatch:

```
START ....
Process exited before completing request 
END ....
```
If I try to run the Lambda from the AWS Console, I get this message as result:
```
{"cause":null,"stackTrace":[],"message":"An error occurred and the request cannot be processed. (Service: AWSLambda; Status Code: 500; Error Code: ServiceException; Request ID: d8b716ee-a223-11e4-acf1-7d60fa88079a)","fullmessage":null,"status":500,"localizedMessage":"An error occurred and the request cannot be processed. (Service: AWSLambda; Status Code: 500; Error Code: ServiceException; Request ID: d8b716ee-a223-11e4-acf1-7d60fa88079a)","suppressed":[]}
```

I'm currently using IAM Credentials with Administrator access (so it should not be a permission problem) and region "eu-west-1".
Also I needed to specify a Role (I think this is an Execution Role,  right?) If I don't I get "MissingRequiredParameter: Missing required key 'Role' in params", even if the comment says "not necessary if user has full access".

Thank you,

**Comment by slvnperron** [comment]
Hi @lusentis,
I am sorry, there's an error in the documentation. There should always be a role specified.
Please attach a role with the following policies:
```AmazonSQSFullAccess
AmazonS3FullAccess```
Let me know if this works for you

**Comment by lusentis** [comment]
Thankyou! Now everything works :+1:

**Comment by contractorwolf** [comment]
I am getting something similar trying to run an AWS Lambda from inside an AWS Step Function. Lambda Service Exception: An error occurred and the request cannot be processed. (Service: AWSLambda; Status Code: 500; Error Code: ServiceException; Request ID: {removed}).  This had also worked previously, and it doesnt seem to hit the Lambda at all (nothing in cloudwatch from the attempt), and now seems to be working again with no changes, but I need to understand what could be causing the issue.  Is it def. a permissions error?

**Comment by deoresheetal** [comment]
I am getting same error. Any updates on this?

**Comment by contractorwolf** [comment]
@deoresheetal I am sorry, it has been a while since I hit that issue and I cant recall exactly what I did to solve it.  I went back and looked and I think it was a permssions issue relating to the "IAM role for executions" that was being used on the step function.  Make sure that has the correct permissions to execute the Lambda (AWSLambdaRole).  Let me know if that worked or if you were able to figure it out in case someone else googles the issue and gets to this page.
